idaes/generic_models/properties/core/generic/generic_property.py

```diff
@@ -833,11 +833,15 @@ def define_metadata(cls, obj):
         # created by state var methods
         obj.add_properties(
             {'flow_mol': {'method': None},
-             'flow_mol_phase': {'method': None},
```

Comment:
From my opening comments, I would suggest that `flow_mol_phase_comp` and `mole_frac_phase_comp` should have `'method': None` based on the fact that the state definition module is expected to construct these. Thus, they should not have build-on-demand methods available to them (or at least, the state definition module can set up the build-on-demand methods if desired).

However, `flow_mol` should be given a build-on-demand method based on the sum of `flow_mol_phase_comp`.

Reply:
Forgot what I was doing once I got into it.  Should be fixed.